### ISSUE TITLE
Conflict-free initialization of Materialize JS

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#31]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

Ideally, a custom JavaScript namespace that initializes [Materialize](https://materializecss.com/).

### Why

We need to be sure that the Materialize JavaScript initialization code does not interfere with any other library or code.

### Issue expands upon

This issue expands upon another issue(s): #29 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
